ui/app/services/capabilities.ts

```diff
@@ -40,14 +40,13 @@ export default class CapabilitiesService extends Service {
     return assert('query object must contain "paths" or "path" key', false);
   }
 
-  async fetchMultiplePaths(paths: string[]): MultipleCapabilities | AdapterError {
```

Comment:
Some TS cleanup -- async/await is just syntactic sugar on Promise, so async methods always return a `Promise<>`